example/main.tsx

```diff
@@ -71,23 +71,13 @@ const App = () => {
     <div className="app">
       <GitHubCorner />
       <h1> React Tags Example </h1>
-      <div className="fuzzySearch_container">
-        <button
-          className="fuzzySearch_toggleCta"
-          onClick={() => setFuzzySearch(!fuzzySearch)}>
-          Fuzzy search on suggestions is {fuzzySearch ? 'on' : 'off'}
-        </button>
-        {fuzzySearch ? (
-          <label>
-            current fuzzy distance is: {fuzzyDistance}
-            <input
-              className="fuzzySearch_distanceInput"
-              placeholder="update fuzzy distance"
-              onChange={(e) => setFuzzyDistance(+e.target.value)}
-            />
-          </label>
-        ) : null}
-      </div>
+
+      <button
+        className="fuzzySearch_toggleCta"
+        onClick={() => setFuzzySearch(!fuzzySearch)}>
+        Fuzzy search on suggestions is {fuzzySearch ? 'on' : 'off'}
+      </button>
```

Comment:
lets make this a check box instead as its a on/off feature

Reply:
done https://github.com/react-tags/react-tags/pull/985/commits/0e4de76a8ae12e8b3b9f676ab80f74116eb53ee1
![image](https://github.com/user-attachments/assets/7ea4104f-631e-4d57-bba7-7197e23b524e)